client/components/migration/feature-announcement.jsx

```diff
@@ -196,6 +128,7 @@ const FeatureAnnouncement = ({ translate, isEligable }) => {
 
 const mapStateToProps = (state, { siteId }) => ({
 	isEligable: isEligableToMigrate(state, siteId),
+	previousMigrationState: wcshippingMigrationState(state, siteId),
```

Comment:
The existing code doesn't have proper formatting, but anyway... :smile: 

```suggestion
	previousMigrationState: wcshippingMigrationState( state, siteId ),
```

Reply:
Hey @Ferdev! Thanks for catching these. How do you set up your `eslint` against this repo's rules? When I run `npm run eslint`, it doesn't say anything. This repo's eslintrc file https://github.com/Automattic/woocommerce-services/blob/trunk/.eslintrc.js also seems different.

I tried `npm install --save-dev @woocommerce/eslint-plugin` but that run properly in this repo.

I want to make sure we don't miss these eslint checks but the repo's precommit hooks don't seem to catch these at all. Thoughts?